platforms/documentation/docs-asciidoctor-extensions-base/src/main/resources/clipboard.css

```
@@ -0,0 +1,29 @@
+/* Custom clipboard CSS code */
```

```suggestion
/* Custom CSS on top of external clipboard.js */
```

Let's make it clear what this builds on